lib/project_types/rails/messages/messages.rb

```diff
@@ -45,12 +45,12 @@ module Messages
             MSG
             dir_exists: "Project directory %s already exists. Please use a different name.",
             install_failure: "Error installing %s gem",
-            node_required: "node is required to create a rails project. Download at https://nodejs.org/en/download.",
-            node_version_failure: "Failed to get the current node version. Please make sure it is installed as " \
-              "per the instructions at https://nodejs.org/en.",
-            yarn_required: "yarn is required to create a rails project. Download at " \
+            node_required: "Node.js is required to create a Rails project. Download at https://nodejs.org/en/download/.",
+            node_version_failure: "Failed to get the current Node.js version. Please make sure it is installed as " \
+              "per the instructions at https://nodejs.org/en/.",
```

Comment:
Small nit, happens to be the URL people end up on when following the one without a trailing `/`